tests/unit-tests/test_decoder.py

```diff
@@ -22,16 +22,18 @@
 
 from graphstorm.model import (LinkPredictDotDecoder,
                               LinkPredictDistMultDecoder,
-                              LinkPredictWeightedDotDecoder,
-                              LinkPredictWeightedDistMultDecoder,
                               EntityRegression,
                               MLPEFeatEdgeDecoder,
                               LinkPredictContrastiveDotDecoder,
-                              LinkPredictContrastiveDistMultDecoder)
+                              LinkPredictContrastiveDistMultDecoder,
+                              LinkPredictRotatEDecoder,
+                              LinkPredictContrastiveRotatEDecoder)
 from graphstorm.dataloading import (BUILTIN_LP_UNIFORM_NEG_SAMPLER,
                                     BUILTIN_LP_JOINT_NEG_SAMPLER)
-from graphstorm.eval.utils import calc_distmult_pos_score
+from graphstorm.eval.utils import (calc_distmult_pos_score,
+                                   calc_rotate_pos_score)
```

Comment:
need to test for `calc_distmult_neg_head_score`, `calc_distmult_neg_tail_score`, `calc_rotate_neg_head_score`, and `calc_rotate_neg_tail_score`.

Reply:
They are covered in check_calc_test_scores_rotate_joint_neg and check_calc_test_scores_rotate_uniform_neg